src/core/include/openvino/op/fake_convert.hpp

```diff
@@ -32,7 +32,7 @@ class OPENVINO_API FakeConvert : public Op {
     const std::string& get_destination_type() const;
 
 private:
-    void validate_type() const;
+    void validate_destination_type() const;
```

Comment:
Just question why this function is member function and not free function implemented only in `.cpp`?
This is only for consider for future if would be better to remove it from class.

Reply:
Can be a free function as well, if a class method it can use `m_destination_type` member directly, otherwise `destination_type` need to be passed to the function.

Reply:
yes and is not an issue as you can  check member value or e.g. tensor type using same way.